versions/1.0.0.md

```diff
@@ -113,7 +113,7 @@ Field Name | Type | Description
 ---|:---:|---
 <a name="actionTarget"></a>target | `string` | **REQUIRED** A JSONPath query expression referencing the target objects in the target document.
 <a name="actionDescription"></a>description | `string` | A description of the action. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
-<a name="actionUpdate"></a>update | Any | An object with the properties and values to be merged with the object(s) referenced by the `target`. This property has no impact if `remove` property is `true`.
+<a name="actionUpdate"></a>update | Any | If the `target` of this action object is an object, this field should be an object with the properties and values to merge with the `target`. If the `target` of this action is an array, this field should be an entry to append to the array. To replace an array in its entirety, apply a remove action followed by an update action. This field has no impact if the `remove` field of this action object is `true`.
```

Comment:
> To replace an array in its entirety, apply a remove action followed by an update action.

This can work if the target is an OAS-defined fixed field or a JSON-Schema-defined keyword and the tool knows whether the target is array- or object-typed.

Or if the tool "remembers" the before-type of target that it deleted. In which case one can't replace an array with a single object, only with a one-item array containing that object.

Is that a real problem?